node/get-anycloud.js

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env node
+
+const { exec, } = require('child_process');
+const path = require('path');
+
+const anycloudUrlBase = 'https://github.com/alantech/anycloud/releases/download/v0.1.1/';
```

Comment:
`latest` or we will be in the same situation for the docs? unless we are trying this version to match the one in package.json, if so can we do something like 
```
const pjson = require('./package.json');
const anycloudUrlBase = `https://github.com/alantech/anycloud/releases/download/${pjson.version}/`
```

Reply:
Yes, going to try for version matching after that. Will switch to that. (If you choose to install version x.y.z it should give you that version of the binary, I think)